metricbeat/module/consul/agent/_meta/fields.yml

```diff
@@ -0,0 +1,60 @@
+- name: agent
+  type: group
+  release: beta
+  description: >
+    Agent Metricset fetches metrics information from a Consul instance running as Agent
+  fields:
+    - name: autopilot
+      type: group
+      fields:
+        - name: healthy
+          type: long
+          description: Overall health of the local server cluster. If all servers are considered healthy by Autopilot, this will be set to 1. If any are unhealthy, this will be 0.
```

Comment:
We could convert this to true/false, in consul all metrics are numeric, but we have booleans too :slightly_smiling_face: